lukasz-lapczyk/recruitment-web-app
# Add admin panel

This closes #9 
This fixes #11, fixes #12 

# Overview
Added admin panel with form for creating and editing users.

# Testing prerequisites
- bundle install
- db migrations
- db seeds

# Testing
- navigate to root directory, log in as user_1@example.org, password: Secret19
- [ ] link to admin panel should not be visible on navbar
- log out
- navigate to root directory, log in as admin@example.org, password: Secret09
- [ ] link to admin panel should be visible on navbar
- click on 'Admin panel' link
- [ ] 'New user' button should be visible
- click on 'New user' button
- [ ] form for creating new user should be visible
- [ ] adding/removing fields for specifying interests should be possible without reloading the page
- fill in form with some valid data and click 'Create' button
- [ ] flash message should appear
- [ ] new user should appear on listing
- click on 'Admin panel' link
- choose some user to edit by clicking proper icon next to user on listing
- [ ] form for editing user should be visible
- fill in form with new valid values
- click on 'Update' button
- [ ] flash message should appear
- [ ] edited user should appear on listing with new data
- log out
- navigate to root directory
- [ ] log in with e-mail of user created previously and password: secret
